### PR TITLE
Original Pull Request  After updation

### PR DESCRIPTION
**Pull request by hamadhasan9**

### CODE REVIEW

File: FRONTEND/src/components/gallery/Gallery.jsx [hunk 1]
```diff
@@ -1,65 +1,51 @@
...
+
 const Gallery = () => {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
+    fetchRandomImages();
+  }, []); // Fetch random images when the component mounts
+
+  const fetchRandomImages = () => {
+    const apiUrl = "https://source.unsplash.com/random/800x600?advertising";
```
Comment (okusjid) [inline FRONTEND/src/components/gallery/Gallery.jsx]: Change the url, so it fetches only bill-board related images

File: FRONTEND/src/components/gallery/Gallery.jsx [hunk 2]
```diff
@@ -1,65 +1,51 @@
...
-import c from "../../assets/Gallery/c.jpeg"; // Update the path
-import image8 from "../../assets/Gallery/image8.jpeg"; // Update the path
-import image9 from "../../assets/Gallery/image9.jpeg"; // Update the path
+import React, { useState, useEffect } from "react";
+import "./Gallery.css"; // Import CSS file for animations
+
 const Gallery = () => {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
```
Comment (okusjid) [inline FRONTEND/src/components/gallery/Gallery.jsx:8]: It should remain the same unless the page reloads. But, it is causing trouble when I scroll down

File: FRONTEND/src/components/About/top.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                About AdvisioScreens
+              </h2>
+              <h2 className="mb-6 max-w-lg text-5xl font-light leading-snug tracking-tight text-g1 sm:text-7xl sm:leading-snug">
+                We make you look{" "}
+                <span className="my-1 inline-block border-b-8 border-g4 bg-white px-4 font-bold text-g4 animate__animated animate__flash">
+                  different
+                </span>
+              </h2>
+              <p className="text-base text-gray-900">
+                Sed ut perspiciatis unde omnis iste natus error sit voluptatem
```
Comment (okusjid) [inline FRONTEND/src/components/About/top.jsx]: Cant Understand the use of this ?! :P

File: FRONTEND/src/components/About/ValuesSection.jsx [hunk 4]
```diff
@@ -31,7 +33,9 @@ const ValuesSection = () => {
           {/* Benefit 3 */}
           <div className="p-6 bg-white rounded-lg shadow-md hover:shadow-lg transition-shadow duration-300">
             <FaRegChartBar className="text-4xl text-blue-600 mx-auto mb-4" />
-            <h3 className="text-2xl font-semibold mb-3">Make out-of-Home easy.</h3>
+            <h3 className="text-2xl font-semibold mb-3">
+              Make out-of-Home easy.
```
Comment (okusjid) [inline FRONTEND/src/components/About/ValuesSection.jsx]: Why are there "." full stops in some values? Make it consistent by removing them

File: FRONTEND/src/components/About/ByNumber.jsx
Comment (okusjid) [inline FRONTEND/src/components/About/ByNumber.jsx:1]: These numbers differ with that of home page. Please update

File: FRONTEND/src/routes/index.jsx [hunk 5]
```diff
@@ -34,9 +35,10 @@ const Routing = () => {
             <Route path="/error" element={<ErrorPage/>}/>
             <Route path='/howitworks' element={<Howitworks />} />
             <Route path='/privacy' element={<Privacy />} />
+            <Route path='/Gallery' element={<Gallery/>} />
             {/* Protect this route or any other routes that require authentication */}
             {/* <Route 
-              path='/media-owners' 
+              path='/media-ownegrs'
```
Comment (okusjid) [inline FRONTEND/src/routes/index.jsx]: Why play around with path!?

File: FRONTEND/src/components/gallery/Gallery.jsx
Comment (okusjid) [inline FRONTEND/src/components/gallery/Gallery.jsx:1]: Fix the fetching issue as it loads again if I scroll down. 
Change url so it fetches bill-board related pics.

File: FRONTEND/src/components/Privacy Policy/Privacy.jsx
Comment (okusjid) [inline FRONTEND/src/components/Privacy Policy/Privacy.jsx:1]: Remove this page. We dont need privacy policy,

**Review by okusjid — CHANGES_REQUESTED** [review]

File: FRONTEND/src/components/gallery/Gallery.jsx (hunk 1 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx]: const apiUrl = "https://source.unsplash.com/random/800x600?billboard";
   Changed

File: FRONTEND/src/components/gallery/Gallery.jsx (hunk 2 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx:8]: Resolved No reloads on Scroll Down

File: FRONTEND/src/components/About/ValuesSection.jsx (hunk 4 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/About/ValuesSection.jsx]: Done

File: FRONTEND/src/components/About/ByNumber.jsx
Comment (hamadhasan9) [reply FRONTEND/src/components/About/ByNumber.jsx:1]: Only compians Numbers can match to the Projects others Numbers like Impression that are not on Home page will remian different

File: FRONTEND/src/routes/index.jsx (hunk 5 above)
Comment (hamadhasan9) [reply FRONTEND/src/routes/index.jsx]: updated

File: FRONTEND/src/components/gallery/Gallery.jsx
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx:1]: resolved

File: FRONTEND/src/components/Privacy Policy/Privacy.jsx
Comment (hamadhasan9) [reply FRONTEND/src/components/Privacy Policy/Privacy.jsx:1]: okay i will

Comment (hamadhasan9) [reply FRONTEND/src/components/Privacy Policy/Privacy.jsx:1]: Removeed

File: FRONTEND/src/components/gallery/Gallery.jsx
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx:1]: Done

File: FRONTEND/src/routes/index.jsx (hunk 5 above)
Comment (hamadhasan9) [reply FRONTEND/src/routes/index.jsx]: ???? that  can be typed error and missed to resovled beacuse that is commented part

File: FRONTEND/src/components/About/ByNumber.jsx
Comment (hamadhasan9) [reply FRONTEND/src/components/About/ByNumber.jsx:1]: updated ... compiagn similar to numbers of projects and customers to golbal partners

File: FRONTEND/src/components/About/ValuesSection.jsx (hunk 4 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/About/ValuesSection.jsx]: Done

File: FRONTEND/src/components/About/top.jsx (hunk 3 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/About/top.jsx]: updated

File: FRONTEND/src/components/gallery/Gallery.jsx (hunk 1 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx]: Done

File: FRONTEND/src/components/gallery/Gallery.jsx (hunk 2 above)
Comment (hamadhasan9) [reply FRONTEND/src/components/gallery/Gallery.jsx:8]: okay its done

**Comment by hamadhasan9** [comment]
Now check and let me know

**Pull request merged**